_posts/2022-05-13-tag-youre-it.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: post
+title:  "Building My Own Content Compass"
+authors:
+- nateboot
+
+date: 2022-05-13
+categories:
+ - intro
+---
+
+## Tag, You’re It! 
+
+I feel a personal and vested interest in ensuring that the forums here at OpenSearch are an information-rich environment. They’re a great tool for directly interacting with users and also for seeing first hand what people need help with. To best leverage them as a tool, I need to generate some actionable data from the forum threads that can be a compass for content generation, and I’d also love for everyone to see what I’m up to.
+
+Tags have been appearing on forum posts retroactively almost as far back as the beginning of the year. I feel no shame here in admitting that it was a boring, lifeless toil for me, all this tagging by hand (let me know if you think I’ve tagged your post incorrectly.) I’ve done my best to guess the intent of any new posts, but I don’t think I can get it right every time. So here’s what we can all do: **tag your posts in the forum**. Help us target the right areas of learning content. My part will be faithfully adding tags to threads that don’t have any. 
```

Comment:
suggested edit : 

"I feel no shame here in admitting that it was a boring, lifeless toil for me, all this tagging by hand (let me know if you think I’ve tagged your post incorrectly.)" 

to: 

I'll admit, this was a cumbersome task at times, but we felt it would greatly benefit the community so I was up for the task! If I got anything wrong, please feel free to edit the tags on your own forum topics. In fact, we welcome your input on this.